test/GovernorCountingFractional.t.sol

```diff
@@ -377,6 +377,7 @@ contract GovernorCountingFractionalTest is Test {
     uint256 _privateKey;
     (_voter.addr, _privateKey) = makeAddrAndKey("voter");
     vm.assume(_voter.addr != address(this));
+    uint128 _initNonce = governor.nonces(_voter.addr);
```

Comment:
This results in always fuzzing with a nonce of 0, we should consider adding a `nonce` input arg then using the StdStorage lib to overwrite the expected initial nonce

Reply:
Yup, good idea. I decided to just expose a setter function on the mock governor to set the nonce. Done in 4067f4b